client/redux/reducers/story.ts

```diff
@@ -0,0 +1,43 @@
+import { Story } from '~components/projects/types';
+
+interface StoryAction {
+  type: 'SET_STORY';
+  story: Story;
+  storyPosition: number;
+}
+
+const initialState = {
+  story: {
+    comments: null,
+    description: null,
+    estimate: null,
+    id: null,
+    name: null,
+    storyType: null,
+    tasks: null,
+    labels: null,
+  },
+  storyPosition: null,
+};
+
+export const story = (state = initialState, action: StoryAction) => {
+  switch (action.type) {
+    case 'SET_STORY':
+      return {
+        ...state,
+        story: {
```

Comment:
since we're copying fields 1:1 we can replace the entire block with just `story: { ...action.story }`